code/style.css

```diff
@@ -1,7 +1,465 @@
+  max-height: 1080px;
+  overflow-x: hidden;
+  /* overflow-x can be used to decide what to do with content that overflows at the left and right edges */
+}
+
+/* add a heading on top of the video in the header */
+.header-content {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  position: fixed;
+  background: linear-gradient(rgba(117, 19, 93, 0.73),rgba(245, 246, 252, 0.52));
+  color: #f1f1f1;
+  width: 100%;
+  height: 100%;
+  max-height: 1080px;
+  font-family: 'Monoton', cursive;
+  text-align: center;
+}
+
+.header-content h1 {
+  font-size: 4.2rem;
+  font-weight: 400;
+}
+
+/* style the video and include a fallback image if the video does not load */
+video#headerVideo {
+  position: fixed;
+  top: 0;
+  left: 0;
+  object-fit: cover;
+  max-height: 1080px;
+  background-image: url(./xavi-cabrera-kn-UmDZQDjM-unsplash-mobile.jpg);
+}
+
+/* style the page content */
+
+article {
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  background: #fff;
+  width: 100vw;
+  font-size: 1.4rem;
+  position: relative;
+  margin: 0;
+}
+
+.text-section p {
+  background: #f0f9fa;
+  font-size: 1.6rem;
+  text-align: center;
+  padding: 60px 60px 60px 60px;
+  margin: 0;
+}
+
+.block-section {
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  background: #fff;
+  padding: 20px 20px 80px 20px;
+}
+
+.block-section h2 {
+  text-align: center;
+  font-size: 1.8rem;
+}
+
+/* accordion */
+
+/* Code adapted from Webdevtrick ( https://webdevtrick.com ) */
+
+.accordion {
+  padding-top: 40px;
+}
+
+.accordion a {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  color: #444;
+  border-bottom: 1px solid #ccc;
+  cursor: pointer;
+  padding: 1.4em;
+  text-align: left;
+  outline: none;
+}
+
+.accordion a:hover,
+.accordion a:hover::after {
+  background-color: #ccc; 
+}
+
+.accordion a.active {
+  background-color: #ccc;
+}
+
+.accordion a::after {
+  font-family: 'Ionicons';
+  content: '\f218';
+  position: absolute;
+  float: right;
+  right: 1rem;
+  color: #DB398D;
+  padding: 5px;
+  width: 30px;
+  height: 30px;
+  border-radius: 50%;
+  border: 1px solid #DB398D;
+  text-align: center;
+}
+
+.accordion a.active::after {
+  font-family: 'Ionicons';
+  content: '\f209';
+}
+
+.accordion .accordion-content {
+  padding: 0 18px;
+  background-color: white;
+  overflow: hidden;
+  display: none;
+  padding-block-start: 1em;
+  padding-block-end: 1em;
+  margin-block-end: 1em;
+  transition: all 0.2s ease 0.15s;
+}
+
+.accordion .accordion-content.active {
+  display: block;
+  transition: all 0.35s ease 0.15s;
+}
+
+/* form */
+
+.form-container {
+  background-color: #00c2db;
+  padding: 0 2em;
+  border: 1px solid #ccc  ;
+  color: #fff;
+}
+
+form {
+  /* Form outline */
+  width: 100%;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+form li + li {
+  margin-top: 1em;
+}
```

Comment:
Is there a specific reason why you chose to use lists for the form? I think you would have more freedom to style (or easily change your styling) if you styled with class instead.